cmd/common.go

```diff
@@ -378,19 +383,64 @@ func Expand(s string) (string, error) {
 	return filepath.Abs(s)
 }
 
-func PushAccount(u string, accountjwt []byte) ([]byte, error) {
+func PushAccount(u string, accountjwt []byte) (int, []byte, error) {
 	resp, err := http.Post(u, "application/text", bytes.NewReader(accountjwt))
 	if err != nil {
-		return nil, err
+		return 0, nil, err
 	}
 	defer resp.Body.Close()
 	data, err := ioutil.ReadAll(resp.Body)
-	if resp.StatusCode != 200 {
-		m := ""
-		if data != nil {
-			m = string(data)
+	return resp.StatusCode, data, err
+}
+
+func IsAccountAvailable(status int) bool {
+	return status == http.StatusOK
+}
+
+func IsAccountPending(status int) bool {
+	return status > http.StatusOK && status < 300
+}
+
+// Validate an operator name
+func OperatorNameValidator(v string) error {
+	operators := GetConfig().ListOperators()
+	for _, o := range operators {
+		if o == v {
+			r := GetConfig().StoreRoot
+			return fmt.Errorf("an operator named %q already exists in %q - specify a different directory with --dir", v, r)
 		}
-		return nil, fmt.Errorf("%s:\n%s", resp.Status, m)
 	}
-	return data, err
+	return nil
+}
+
+func AccountJwtURLFromString(asu string, accountSubject string) (string, error) {
+	u, err := url.Parse(asu)
+	if err != nil {
+		return "", err
+	}
+	u.Path = path.Join(u.Path, "accounts", accountSubject)
+	return u.String(), nil
+}
+
+func AccountJwtURL(oc *jwt.OperatorClaims, ac *jwt.AccountClaims) (string, error) {
```

Comment:
can you add doc to the JWT library for what you expect on these urls - also maybe prompt the correct form if someone is editing an operator jwt to have them

Reply:
https://github.com/nats-io/jwt/pull/50